packages/eyes.sdk/src/EyesBase.js

```diff
@@ -150,6 +150,16 @@
         });
     };
 
+    var _createRenderingInfoWithCache = function (createRenderingInfo) {
```

Comment:
Just a small comment, also regarding my implementation in Selenium 4 - this function just caches. It doesn't have any knowledge about the implementation/input/output. I'm pretty sure lodash has some memoize function for this, and we probably can also extract this into a utility function. (not here, in the monorepo)